queries.sql

```diff
@@ -16,4 +16,68 @@ SELECT * FROM animals WHERE name='Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon'
+WHERE name LIKE '%mon%';
+
+UPDATE animals SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
+
+SELECT name, species FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+ DELETE FROM animals
+ WHERE date_of_birth > '01-01-2022';
+ SAVEPOINT SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+ROLLBACK TO SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+COMMIT;
+
+SELECT count(*) FROM animals;
+
+SELECT count(*) FROM animals
+WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg)
+FROM animals;
+
+SELECT 
+  MAX(escape_attempts)
+FROM animals
+WHERE neutered = TRUE OR neutered = FASLE;
```

Comment:
# [OPTIONAL] 
* This `FASLE` is spelt wrongly and as a result, it throws an error kindly fix it 
* Please ensure you always run your queries just to the sure they are error-free 🙏